client/src/Content/firstPage.js

```diff
@@ -0,0 +1,54 @@
+export default function firstContent() {
+	let Content = [
+		{
+			id: 1,
+			header: 'Personal Info',
+			body:
+				'Please fill your personal information before get started with the next applications steps',
+		},
+		{
+			id: 2,
+			header: 'Typing',
+			body:
+				'Typing is very important skill in life and especially in coding. If you are a good typist you will find learning to code much, much easier.\n' +
+				'\n' +
+				'Join the GSG classroom on TypingClub, using the same email address you used in you GSG Code Academy application. If you do not use the same email address we will not know your score.\n' +
+				'Click at the following link to join the classroom.\n' +
+				'(https://gsg-code-academy.typingclub.com/signup/GMUR84A)' +
+				'\n' +
+				'\n' +
+				'if the link above is not working then follow these steps:' +
+				'\n' +
+				'\n' +
+				'1- First click this link to join the class' +
+				'\n' +
+				'https://gsg-code-academy.typingclub.com/' +
+				'\n' +
+				'2- Click Enter a Class Code \n' +
+				'3- Copy and paste this code (GMUR84A) then click Sign up after that you can fill in your information\n' +
+				'\n' +
+				'By the application deadline, you must complete at least the first 50 lessons in Typing Club.\n' +
+				'Each lesson should take less than 10 minutes to\n' +
+				'complete. The best way to improve your typing is to do a small amount, practising every day, rather than doing all of the lessons at once.',
```

Comment:
I wonder if template literals would work? They might alleviate the need for the `\n` action.

Reply:
whats that ?